### ISSUE TITLE
mouseleave not firing?

### ISSUE THREAD

**Issue by wiber** [furf/jquery-ui-touch-punch#81]
asked a question here: http://stackoverflow.com/questions/16021562/triggering-mouseleave-approximating-touchleave

Unsure where the issue lies, but could be that this plugin isn't firing mouseleave the same way as desktop.

**Comment by rvignacio** [comment]
The question was deleted... do you still have this problem? Because I'm struggling with something similar.

**Comment by yertizz** [comment]
Yes. But can't wait any longer. Dismal failure getting a real working response.
Phil
  ----- Original Message -----

**Comment by rvignacio** [comment]
Thanks @yertizz, I've read the code and the plugin doesn't trigger a touchleave event...

Sorry, one last question, does anybody know which is the state of this project? (@furf?) Are pull requests going to be accepted? Because there are a lot of them out there and I would like to work with updated code to avoid merge conflicts, etc...

**Edit:** found the problem, it isn't triggering a touchleave event because it was [removed from the spec](http://www.w3.org/TR/touch-events/#list-of-touchevent-types).
